tests/CMakeLists.txt

```diff
@@ -34,3 +34,4 @@ get_property(STL_LIT_TEST_DIRS GLOBAL PROPERTY STL_LIT_TEST_DIRS)
 
 add_custom_target(STL-CI COMMAND ${STL_LIT_COMMAND} -Dnotags=ASAN ${STL_LIT_TEST_DIRS} USES_TERMINAL)
 add_custom_target(STL-ASan-CI COMMAND ${STL_LIT_COMMAND} -Dtags=ASAN ${STL_LIT_TEST_DIRS} USES_TERMINAL)
+add_custom_target(STL-only-edg-tests COMMAND ${STL_LIT_COMMAND} -Dnotags=ASAN -Dedg_only=True --xunit-xml-output test-results.xml ${STL_LIT_TEST_DIRS} USES_TERMINAL)

```

Comment:
(No change requested) FWIW the existing targets have `STL-` prefixes only because we decided to make them match the names of the pipelines that use them. You could name this something shorter like `only-edg-tests` or `test-only-edg` if you prefer.

Reply:
I changed the target name and flag to be consistent.